src/textual/color.py

```diff
@@ -145,6 +146,11 @@ def is_transparent(self) -> bool:
         """Check if the color is transparent, i.e. has 0 alpha."""
         return self.a == 0
 
+    @property
+    def is_auto(self) -> bool:
+        """Check if the color is "auto"."""
+        return self.r == AUTO.r and self.g == AUTO.g and self.b == AUTO.b
```

Comment:
Would probably be more elegant with `(self.r, self.g, self.b) == (AUTO.r, AUTO.g, AUTO.b)`, but since we're aiming at optimising for performance in most cases I guess creating a tuple takes a bit more time than doing this triple equality condition? :slightly_smiling_face:

Reply:
`self[:3] == AUTO[:3]` is probably fastest. But I'm afraid I'm going to veto this sentinel color object.